Cargo.toml

```diff
@@ -17,7 +17,10 @@ tracing-log = "0.2.0"
 tracing-bunyan-formatter = "0.3.10"
 tracing-appender = "0.2.3"
 tonic = { version = "0.12.3", features = ["gzip", "tls", "tls-roots", "tls-native-roots"] }
-opentelemetry-proto = { version = "0.27.0" }
+opentelemetry = "0.29.0"
+opentelemetry-otlp = { version = "0.29.0"}
```

Comment:
Is the otlp crate used explicitly or is this needed for a dep? I didn't see a direct `use` statement, so wasn't sure.